src/__tests__/develop.spec.js

```diff
@@ -0,0 +1,22 @@
+import GetNumberSum from '../develop.js'
+function test() {
+  const testCases = [
+    { input: { a: 1, b: 2 }, expected: 3 },
+    { input: { a: -2, b: 1 }, expected: 0 },
+    { input: { a: 'not a number', b: 'number' }, expected: "Error: Cannot plus non number params" } 
+  ];
+  let testResults
+  try {
+    testResults = testCases.map((testCase) => {
+    const result = GetNumberSum(testCase.input.a, testCase.input.b); 
+    const isPassed = result === testCase.expected; 
+    return isPassed;
+  });
+} catch (error) {
+  console.log('err')
+}
+
+    return testResults;
+  }
+  const results = test();
+  console.log(results);
```

Comment:
```suggestion
  let testResults
    testResults = testCases.map((testCase) => {
    const result = GetNumberSum(testCase.input.a, testCase.input.b); 
    const isPassed = result === testCase.expected; 
    return isPassed;
  });
    return testResults;
  }
  try{
  const results = test();
  //add your implementation to throw an error when some test is failed
  }catch(error){
  cosnole.log(error)
  }
  console.log(results);
```

Reply:
ty

Reply:
![Снимок экрана 2024-04-30 100710](https://github.com/Octarchy/Algorythm_learning/assets/166847182/85c88371-d1cf-4800-b258-c89ac9f8994e)
ok, I tried to do the same as you, but my block does not work and does not display an error

Reply:
Sure it's not. Because you didnt throw any error.
Why you expecting that error occures if you just run code and didnt add any statements for errors passing :smile:

Reply:
then what should be the error?